test/core/integration/concrete/lockup/renounce/renounce.t.sol

```diff
@@ -57,16 +41,6 @@ abstract contract Renounce_Integration_Concrete_Test is Integration_Test, Lockup
         lockup.renounce(defaultStreamId);
     }
 
-    /// @dev This modifier runs the test twice: once with a "PENDING" status, and once with a "STREAMING" status.
-    modifier givenWarmStream() {
-        vm.warp({ newTimestamp: getBlockTimestamp() - 1 seconds });
```

Comment:
I can see that the code of `givenWarmStream` in `Modifiers` contract does not contain the same implementation as this. Was this redundant? If no, then the same logic should also be present in the modifiers contract.

Reply:
removed by mistake, as there were places where this modifier didn't have this logic:

https://github.com/sablier-labs/v2-core/blob/997444866a2f9279ffe4bdde59462e4598d6d87e/test/core/integration/shared/lockup/cancel.t.sol#L38-L40